py/picca/data.py

```diff
@@ -368,14 +369,14 @@ def from_ascii(cls,line):
         m_z = float(a[6])
         m_SNR = float(a[7])
         m_reso = float(a[8])
+        dll = float(a[9])
+        
+        nbpixel = int(a[10])
+        de = sp.array([float(a[11+i]) for i in range(nbpixel)])
```

Comment:
`de = sp.array([a[11+i] for i in range(nbpixel)]).astype(float)` is more elegant and should be faster

Reply:
actually the more compact de = a[11:11+nbpixel].astype(float) would completely avoid the for loop and instantiating a list.